content/content-strategy/define-user-content-needs/create-actionable-artefacts/top-tasks/index.yml

```diff
@@ -0,0 +1,27 @@
+pagetitle: Top tasks
+description: Abstract
+weight: 10
+layout: layout/topic
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
```

Comment:
numbered file convention is hard to follow. Suggestion: use comments, e.g. `# describe file` to ensure that the file structure is correct. Even better: use keyword from section to label filenames instead of just numbers.